### ISSUE TITLE
Could not able to send html template as the email body

### ISSUE THREAD

**Issue by suganyasuven** [ballerina-platform/ballerina-standard-library#2778]
**Description:**
$title
When we try to set the html template as mail body, email service sends with empty body.

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by ayeshLK** [comment]
This issue is reproducible with following code-segment:
```ballerina
string htmlBody = check io:fileReadString("_resources/welcome-new-org.html");

email:Message email = {
    to: "user2",
    subject: "Hello, World!",
    htmlBody: htmlBody,
    'from: "author@email.com"
};

check smtpClientEp->sendMessage(email);
```

**Comment by ayeshLK** [comment]
Fixed with [#466](https://github.com/ballerina-platform/module-ballerina-email/pull/466)
